README.md

```diff
@@ -10,6 +10,7 @@ The `develop` branch should be on recent freezes, and the `feature/latest-ci` br
 
    Version |          FHIR | &nbsp;
 -----------|---------------|---------
+ **5.0.0** |               | (R4)
```

Comment:
Should this be `(R5)`?

Reply:
No, it supports R4 and will later add STU3 and DSTU3. I bumped the major version since repackaging could be a breaking change for some.